packages/core/src/layout/SignInPage/SignInPage.tsx

```diff
@@ -52,7 +54,7 @@ export const SignInPage: FC<Props> = ({
   }
 
   return (
-    <Page>
+    <Page theme={backstageTheme.getPageTheme({ themeId: 'home' })}>
```

Comment:
I'm thinking we can make lives easier for the consumers of the `Page` component by mirroring the input to `getPageTheme` in the `Page` props instead. So that it's used like this instead:

```tsx
<Page themeId="home">
```

If we in the future need to add more parameters to `getPageTheme`, we can also add that as props to `Page`.

Reply:
Realizing that we're also back to square 1 here, so perhaps just change the name from `themeId` to `theme` instead, and we can leave the props for `Page` as they were :grin:

Reply:
Done 😼.